src/Tasks.UnitTests/SecurityUtil_Tests.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.ConstrainedExecution;
+using System.Runtime.Versioning;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Build.Tasks.Deployment.ManifestUtilities;
+using Shouldly;
+using Xunit;
+
+namespace Microsoft.Build.Tasks.UnitTests
+{
+    public class SecurityUtil_Tests
+    {
+        private static string TestAssembliesPaths { get; } = Path.Combine(AppContext.BaseDirectory, "TestResources");
+
+        [WindowsOnlyFact]
+        [SupportedOSPlatform("windows")]
+        public void SignFile_Success()
+        {
+            Uri timestampUrl = new("http://timestamp.comodoca.com/rfc3161");
+            string clickOnceManifest = Path.Combine(TestAssembliesPaths, "ClickOnceProfile.pubxml");
+            string targetFrameworkVersion = Constants.TargetFrameworkVersion40;
+            string targetFrameworkIdentifier = Constants.DotNetFrameworkIdentifier;
```

Comment:
If we're testing for .NET Core, I think target framework identifier should be DotNetCoreAppIdentifier.
If this test is for full FX, TargetFrameworkVersion 4.5 & higher only use SHA256.  TargetVersion40 will use SHA1 I think.